From 7063547d0cefcff3366b9bc4add14f51822993c1 Mon Sep 17 00:00:00 2001
From: retr0h <john@dewey.ws>
Date: Thu, 4 Mar 2010 19:49:20 -0800
Subject: [PATCH] Updated ActionController::Base.asset_host example

The example provided contrived.  It is not useful when wanting to use
1-based asset hosts (since it would generate a random asset host for
the same asset).
---
 .../lib/action_view/helpers/asset_tag_helper.rb    |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/actionpack/lib/action_view/helpers/asset_tag_helper.rb b/actionpack/lib/action_view/helpers/asset_tag_helper.rb
index de3d61e..8215195 100644
--- a/actionpack/lib/action_view/helpers/asset_tag_helper.rb
+++ b/actionpack/lib/action_view/helpers/asset_tag_helper.rb
@@ -58,7 +58,7 @@ module ActionView
     # +asset_host+ to a proc like this:
     #
     #   ActionController::Base.asset_host = Proc.new { |source|
-    #     "http://assets#{rand(2) + 1}.example.com"
+    #     "http://assets#{source.hash % 2 + 1}.example.com"
     #   }
     #   image_tag("rails.png")
     #   # => <img alt="Rails" src="http://assets0.example.com/images/rails.png?1230601161" />
@@ -66,7 +66,7 @@ module ActionView
     #   # => <link href="http://assets1.example.com/stylesheets/application.css?1232285206" media="screen" rel="stylesheet" type="text/css" />
     #
     # The example above generates "http://assets1.example.com" and
-    # "http://assets2.example.com" randomly. This option is useful for example if
+    # "http://assets2.example.com". This option is useful for example if
     # you need fewer/more than four hosts, custom host names, etc.
     #
     # As you see the proc takes a +source+ parameter. That's a string with the
-- 
1.6.0.4

